server.ts

```diff
@@ -26,11 +26,46 @@ app.use(cors()) //add CORS support to each following route handler
 const client = new Client(dbConfig);
 client.connect();
 
-app.get("/", async (req, res) => {
-  const dbres = await client.query('select * from categories');
-  res.json(dbres.rows);
+app.get("/dogs", async (req, res) => {
+  const dbres = await client.query('select breed from dogs');
+  res.set('Access-Control-Allow-Origin', '*')
+  res.status(200).json(dbres.rows);
 });
 
+app.get("/topten", async (req, res) => {
+  const dbres = await client.query(`SELECT * FROM dogs ORDER BY votes DESC, breed ASC LIMIT 10`)
+  res.set('Access-Control-Allow-Origin', '*')
+  res.status(200).json(dbres.rows)
```

Comment:
we see you've used try/catch syntax elsewhere, maybe a good idea  to use it here too + stay consistent with that syntax + it's good practice